contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,287 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::{Request, Response, Rocket};
+
+use super::policy::*;
+
+/// SpaceHelmet implements a [Fairing](../../rocket/fairing/trait.Fairing.html)
+/// that adds http headers to outgoing responses that control security features
+/// on the client.
+///
+/// # Usage
+///
+/// `SpaceHelmet` can be used in several ways.
+///
+/// To use it with its [defaults](#method.default), create a new instance and
+/// attach it to [Rocket](../../rocket/struct.Rocket.html)
+/// as shown.
+/// ```rust
+/// # extern crate rocket;
+/// # extern crate rocket_contrib;
+///
+/// use rocket_contrib::space_helmet::SpaceHelmet;
+///
+/// let helmet = SpaceHelmet::default();
+/// let rocket = rocket::ignite().attach(helmet);
```

Comment:
It would be better style to do `rocket::ignite().attach(SpaceHelmet::default())`.